docs/docs-beta/docs/guides/data-modeling/data-assets.md

```diff
@@ -2,3 +2,111 @@
 title: "Create data assets"
 sidebar_position: 10
 ---
+
+Data assets are at the core of Dagster. By modelling your data as assets you unlock: declarative automation, lineage, and a searchable data catalog. This guide will help you understand how to model your data as assets.
+
+There are three ways of create data assets:
+
+* Using the `@asset` decorator to declare a single asset.
+* Using the `@multi_asset` decorator to output multiple assets from a single operation.
+* Using the `@graph_asset` decorator to output a single asset from multiple operations without making each operation itself an asset.
+
+This guide walks through all three methods.
+
+***
+
+<details>
+  <summary>Prerequisites</summary>
+
+To follow the steps in this guide, you'll need:
+
+* A basic understanding of Dagster concepts such as assets and resources
+* Dagster installed and a working project setup
+
+</details>
+
+***
+
+## Define a single data asset using the `@asset` decorator
+
+The simplest way to define a data asset in Dagster is by using the `@asset` decorator. This decorator marks a Python function as an asset, with the function's output being the data produced by the asset.
+
+```python
+@asset
+def my_data_asset():
+    return {"key": "value"}
+```
+
+In this example, `my_data_asset` is an asset that returns a simple dictionary. Dagster automatically tracks the dependencies of this asset and handles its execution within the pipeline.
+
+The benefits of this approach are:
+
+* The asset is self-contained, which makes the code easier to test and maintain.
+
+The downsides of this approach are:
+
+* It’s not well-suited for cases where multiple related assets need to be produced from a single operation.
+
+## Define multiple data assets using the `@multi_asset` decorator
+
+When you need to generate multiple assets from a single operation, you can use the `@multi_asset` decorator. This allows you to output multiple assets while maintaining a single processing function.
+
+```python
+@multi_asset(
```

Comment:
we should be using AssetSpec here: https://docs.dagster.io/concepts/assets/multi-assets